### ISSUE TITLE
Plugin "@typescript-eslint" was conflicted between ".eslintrc » @nuxt/eslint-config"  and ".eslintrc » ../.eslintrc". 

### ISSUE THREAD

**Issue by ennioVisco** [nuxt/eslint-config#276]
Hello, 

I am in a monorepo setting, and I keep getting this error. 
In essence, I have other parts of my codebase in typescript, and I want to share part of the eslint configs, which I keep in the root (`../.eslintrc`).
Unfortunately, when I extend it in my `nuxt` packages that are using this module, I get the error in the subject of this issue. 
Any ideas about how to properly address this?

**Comment by terwer** [comment]
+1 any ideas?

**Comment by terwer** [comment]
Finally, I give up s share config.It works now

```js
module.exports = {
  root: true,
  extends: ["eslint:recommended", "turbo", "prettier", "@nuxt/eslint-config"],

  plugins: ["prettier"],

  rules: {
    // Note: you must disable the base rule as it can report incorrect errors
    semi: "off",
    quotes: "off",
    "@typescript-eslint/no-this-alias": "off",
    "prettier/prettier": "error",
  },
}
```

**Comment by terwer** [comment]
but I have a new problem.ts in .vue files cannot parsed

```
const recentPosts = reactive({
  posts: <Post[]>[],
})
```

this is ok, But eslint report en error.

**Comment by terwer** [comment]
I changed to 

```
const recentPosts = reactive({
  posts: [] as Post[],
})
```

now it worked

**Comment by ennioVisco** [comment]
It seems this is not a problem anymore. Comment and I'll reopen it.
